doc/fileexplorer.rst

```diff
@@ -1,28 +1,79 @@
-:orphan:
+#####
+Files 
+#####
 
-#############
-File Explorer
-#############
-
-The **File Explorer** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a built-in filesystem and directory browser that allows you to view and filter files according to their type and extension.
+You can also open files with the :doc:`editor` or an external tool, and perform some operations.
 
 .. image:: images/file_explorer/file_explorer_standard.png
    :alt: Spyder File Explorer panel, showing a tree view of files and metadata
 
 |
 
 
-==========================
-Performing file operations
-==========================
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and (on Windows) opening an external file explorer.
+===============
+File Operations
+===============
+
+To browse between the files in your system, you can use the arrows in the top of the pane.
+You can expand/contract the folders displayed in the pane to deploy the files and subfolders hierarchically. 
+Double cliking a folder will open it showing the files on it and changing your working directory.
+
+[GIF showing browsing files and showing changing the working directory]
+
+To open a file in the editor from the file pane, double click over its name.
+If you right click over it, you will see a context menu that allows you to access a number of functions including running scripts, creating, renaming, moving, deleting and showing files in your computer's file manager.
+
+[Screenshot showing context menu]
+
+You can also copy and paste one or several files in the folder where you are located in the pane, or copy their absolute and relative's path in the clipboard to paste it as text.
+
+[GIF showing copy path and pasting it in console]
+
+Additionally, the Files pane allows you to perform basic operations with ``git`` like commiting the changes of a file and browsing the repository from which the file is part of.
+
+[GIF showing commiting change of file and then showing repo]
```

Comment:
This might be rather long, but let's see what you come up with and we can always speed it up or cut parts out.